package.json

```diff
@@ -51,6 +51,7 @@
     "eslint-plugin-simple-import-sort": "7.0.0",
     "jest": "26.6.3",
     "lerna": "4.0.0",
+    "node-gyp": "8.4.1",
```

Comment:
We shouldn't add this here, it should be added to the project that requires node-gyp. Which project are you seeing that is trying to use node-gyp to build?